endpoints/put_test.go

```diff
@@ -607,7 +648,7 @@ func TestBadPayloadSizePutError(t *testing.T) {
 
 func TestInternalPutClientError(t *testing.T) {
 	// Valid request
-	reqBody := "{\"puts\":[{\"type\":\"xml\",\"value\":\"text longer than size limit\"}]}"
+	reqBody := "{\"puts\":[{\"type\":\"xml\",\"value\":\"some data\"}]}"
```

Comment:
Is it no longer important (or was it never important?) for the value data to be longer than the size limit?

Reply:
It was never important. `TestInternalPutClientError` asserts we handle an errors coming from the backend side (maybe even the storage service itself)